inject-kotlin-test/src/main/kotlin/io/micronaut/annotation/processing/test/support/KotlinCompilation.kt

```diff
@@ -57,7 +57,11 @@ class KotlinCompilation : AbstractKotlinCompilation<K2JVMCompilerArguments>() {
 	/** Generate metadata for Java 1.8 reflection on method parameters */
 	var javaParameters: Boolean = false
 
-	/** Use the IR backend */
+    /**
+     * Use the IR backend
+     *
+     * @deprecated Removed in 1.9.20
+     */
```

Comment:
is not a test project? can we not just remove it?

Reply:
You're right...gone